main.nf

```diff
@@ -611,6 +666,19 @@ process run_global_fdr_scoring {
                                                     --out=${scored_osw.baseName}_global_merged.osw \\
                                                     --context=global \\
      """
+    } else {
```

Comment:
do we need some extra parameters for the XGboost? you introduce some for LDA?

Reply:
Yes, good point. in the LDA case I use and change the defaults of the additional parameter pi_0 only for the non-realistic tiny test dataset. It was too small to result in a proper target / decoy scoring, but when setting pi0_lambda to 0 it passed. For the XGBoost this parameter does not exist.